cmd/guacone/cmd/files.go

```diff
@@ -105,6 +111,18 @@ var filesCmd = &cobra.Command{
 			logger.Errorf("unable to register file collector: %v", err)
 		}
 
+		// initialize collectsub client
+		csubClient, err := csub_client.NewClient(opts.csubAddr)
+		if err != nil {
+			logger.Infof("collectsub client initialization failed, this ingestion will not pull in any additional data through the collectsub service: %w", err)
+			csubClient = nil
+		} else {
+			defer csubClient.Close()
+		}
+
+		_, err = csubClient.GetCollectEntries(ctx, nil)
```

Comment:
should you wrap this in `if csubClient != nil` ?

Reply:
oh whoops... that's my debug statement, ill remove it

Reply:
fixed!